ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -51,6 +51,15 @@ tests:
     make lint
   container:
     from: golangci-lint
+- as: unit
+  commands: make test
+  container:
+    from: src
+- as: verify
+  commands: |
+    make verify
```

Comment:
What about having something more sophisticated here?

Let me explain myself, `make verify` is calling other targets:
* `patch` - patch some vendor pkg, this one isn't very interesting
* `fmt` - formatting the code - this is the problematic one
* `vet` - report likely mistakes in packages

I would have think that the CI should fail if the files in the PR are not well formatted, therefor I think we should fail the CI in case the output of `make fmt` has modified some files - that would say that the PR authors need to `make fmt` before he pushes.

WDYT?

Reply:
This is probably something we need to do in SRO's Makefile itself though.

Reply:
This is actually a nice idea, but I too think this should be part of the makefile. `verify` target should be reworked, I will post a PR for that.